cms/static/cms/js/dist/3.5.3/bundle.admin.base.min.js

```diff
@@ -1 +1,5 @@
/^$|\/(?:java|ecma)script/i,Je=/^true\/(.*)/,Ke=/^\s*<!(?:\[CDATA\[|--)|(?:\]\]|--)>\s*$/g,Ze={option:[1,"<select multiple='multiple'>","</select>"],legend:[1,"<fieldset>","</fieldset>"],area:[1,"<map>","</map>"],param:[1,"<object>","</object>"],thead:[1,"<table>","</table>"],tr:[2,"<table><tbody>","</tbody></table>"],col:[2,"<table><tbody></tbody><colgroup>","</colgroup></table>"],td:[3,"<table><tbody><tr>","</tr></tbody></table>"],_default:se.htmlSerialize?[0,"",""]:[1,"X<div>","</div>"]},et=w(be),tt=et.appendChild(be.createElement("div"));Ze.optgroup=Ze.option,Ze.tbody=Ze.tfoot=Ze.colgroup=Ze.caption=Ze.thead,Ze.th=Ze.td,ce.extend({clone:function(e,t,n){var r,o,i,a,s,u=ce.contains(e.ownerDocument,e);if(se.html5Clone||ce.isXMLDoc(e)||!Be.test("<"+e.nodeName+">")?i=e.cloneNode(!0):(tt.innerHTML=e.outerHTML,tt.removeChild(i=tt.firstChild)),!(se.noCloneEvent&&se.noCloneChecked||1!==e.nodeType&&11!==e.nodeType||ce.isXMLDoc(e)))for(r=x(i),s=x(e),a=0;null!=(o=s[a]);++a)r[a]&&O(o,r[a]);if(t)if(n)for(s=s||x(e),r=r||x(i),a=0;null!=(o=s[a]);a++)C(o,r[a]);else C(e,i);return r=x(i,"script"),r.length>0&&k(r,!u&&x(e,"script")),r=s=o=null,i},buildFragment:function(e,t,n,r){for(var o,i,a,s,u,c,l,f=e.length,p=w(t),d=[],h=0;f>h;h++)if((i=e[h])||0===i)if("object"===ce.type(i))ce.merge(d,i.nodeType?[i]:i);else if(Xe.test(i)){for(s=s||p.appendChild(t.createElement("div")),u=(We.exec(i)||["",""])[1].toLowerCase(),l=Ze[u]||Ze._default,s.innerHTML=l[1]+i.replace($e,"<$1></$2>")+l[2],o=l[0];o--;)s=s.lastChild;if(!se.leadingWhitespace&&Qe.test(i)&&d.push(t.createTextNode(Qe.exec(i)[0])),!se.tbody)for(i="table"!==u||Ue.test(i)?"<table>"!==l[1]||Ue.test(i)?0:s:s.firstChild,o=i&&i.childNodes.length;o--;)ce.nodeName(c=i.childNodes[o],"tbody")&&!c.childNodes.length&&i.removeChild(c);for(ce.merge(d,s.childNodes),s.textContent="";s.firstChild;)s.removeChild(s.firstChild);s=p.lastChild}else d.push(t.createTextNode(i));for(s&&p.removeChild(s),se.appendChecked||ce.grep(x(d,"input"),T),h=0;i=d[h++];)if((!r||-1===ce.inArray(i,r))&&(a=ce.contains(i.ownerDocument,i),s=x(p.appendChild(i),"script"),a&&k(s),n))for(o=0;i=s[o++];)Ge.test(i.type||"")&&n.push(i);return s=null,p},cleanData:function(e,t){for(var n,r,o,i,a=0,s=ce.expando,u=ce.cache,c=se.deleteExpando,l=ce.event.special;null!=(n=e[a]);a++)if((t||ce.acceptData(n))&&(o=n[s],i=o&&u[o])){if(i.events)for(r in i.events)l[r]?ce.event.remove(n,r):ce.removeEvent(n,r,i.handle);u[o]&&(delete u[o],c?delete n[s]:typeof n.removeAttribute!==Ce?n.removeAttribute(s):n[s]=null,Z.push(o))}}}),ce.fn.extend({text:function(e){return Me(this,function(e){return void 0===e?ce.text(this):this.empty().append((this[0]&&this[0].ownerDocument||be).createTextNode(e))},null,e,arguments.length)},append:function(){return this.domManip(arguments,function(e){if(1===this.nodeType||11===this.nodeType||9===this.nodeType){S(this,e).appendChild(e)}})},prepend:function(){return this.domManip(arguments,function(e){if(1===this.nodeType||11===this.nodeType||9===this.nodeT
```

Comment:
Bad merge?

Reply:
just need to regenerate them, are you building from the branch @PeterW-LWL ?

Reply:
I just noticed it, reading the diffs. I put this annotation on the new line 5, but github shows it on the old line 1. In the ["File Changed"](https://github.com/divio/django-cms/pull/6646/files#diff-6a124d697d754dd8f31b1ad0877e65e4R5) view you can see what I mean.

I do know that this is a generated file.